openstack/clustering/v1/policies/results.go

```diff
@@ -54,21 +56,57 @@ func (r PolicyPage) LastMarker() (string, error) {
 	return policies[len(policies)-1].ID, nil
 }
 
+const RFC3339WithZ = "2006-01-02T15:04:05Z"
+
 func (r *Policy) UnmarshalJSON(b []byte) error {
 	type tmp Policy
 	var s struct {
```

Comment:
We can handle the unmarshalling of `Version` here. Add it below `UpdatedAt` as:

```go
Version interface{} `json:"version"`
```

And then below the `s.UpdatedAt != ""`  block, you can add logic similar to here:

https://github.com/gophercloud/gophercloud/blob/master/openstack/compute/v2/extensions/floatingips/results.go#L42-L47

Reply:
`Version` is nested inside the `Spec` struct so I couldn't define it after `UpdatedAt` inside `s`.
So I ended up defining `Version` as `interface{}` inside `Spec`.  Do you have any better idea?

Reply:
argh. That's my mistake. I got the two structs (Policy and Spec) mixed up.

We will need a second `UnmarshalJSON` method, but on `Spec` instead of `Version` as the initial implementation.